src/main/kotlin/com/github/mitchellolsthoorn/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.mitchellolsthoorn.testgenie.services
+
+import com.intellij.ide.highlighter.JavaFileType
+import com.intellij.openapi.project.Project
+import com.intellij.psi.JavaCodeFragmentFactory
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.ui.EditorTextField
+import com.intellij.ui.components.JBScrollPane
+import java.awt.BorderLayout
+import java.awt.Dimension
+import javax.swing.*
+
+class TestCaseDisplayService(private val project: Project) {
+
+    val mainPanel: JPanel = JPanel()
+    private val applyButton: JButton = JButton("Apply")
+    private val allTestCasePanel: JPanel = JPanel()
+    private val scrollPane: JBScrollPane = JBScrollPane(allTestCasePanel)
+
+    init {
+        allTestCasePanel.layout = BoxLayout(allTestCasePanel, BoxLayout.Y_AXIS)
+        mainPanel.layout = BorderLayout()
+        mainPanel.add(applyButton, BorderLayout.SOUTH)
+        mainPanel.add(scrollPane, BorderLayout.CENTER)
+    }
+
+    /**
+     * Fill the panel with the generated test cases. Remove all previously shown test cases.
```

Comment:
I tried to see if the old tests are removed, however because they are hard-coded for now to be the same, I can not check if it updates. I trust you that it works, since you said that this is what I should do.